Header header
hough_point[] points
geometry_msgs/Point begin
geometry_msgs/Point end
